src/getInstanceId.js

```diff
@@ -13,14 +14,13 @@ var logger = require("./logger");
 module.exports = function(options) {
   return getFirebaseProject(options.project).then(function(project) {
     if (!_.has(project, "resources.realtimeDatabaseInstance")) {
-      logger.debug(
-        "[WARNING] Unable to fetch default resources. Falling back to project id (" +
-          options.project +
```

Comment:
Might be a good idea to fallback to `<project>-default-rtdb` now.

Reply:
If `<project>-default-rtdb` exists, it will be present in the response for `getFirebaseProject`. I think we should remove this fallback now.